tracing-opentelemetry-instrumentation-sdk/src/http/http_server.rs

```diff
@@ -29,6 +29,7 @@ pub fn make_span_from_request<B>(req: &http::Request<B>) -> tracing::Span {
         trace_id = Empty, // to set on response
         request_id = Empty, // to set
         exception.message = Empty, // to set on response
+        "span.type" = "web",
```

Comment:
Please remove the quote on key (like for other keys).
`span.type = "web"`

Reply:
I tried to do that, but `type` is a rust reserved keyword.
To escape, I added the prefix `r#` like that: `span.r#type = "web"` but then the tests failed because the snapshots were containing the string `"span.r#type"` instead of `span.type` as you can check in the print.
Do you know any way to fix that?

<img width="176" alt="Screenshot 2023-07-26 at 15 44 44" src="https://github.com/davidB/tracing-opentelemetry-instrumentation-sdk/assets/8892624/ccdd50dc-11f4-468e-94be-209d79d1a0fb">